testing/ios/IosUnitTests/build_and_run_tests.sh

```diff
@@ -3,5 +3,6 @@ pushd $PWD
 cd ../../../..
 ./flutter/tools/gn --ios --simulator --unoptimized
 ninja -j 100 -C out/ios_debug_sim_unopt
+ninja -j 100 -C out/ios_debug_sim_unopt ios_test_flutter
```

Comment:
is this out/ios_test_flutter?

Reply:
nope, we are piggy backing on `./flutter/tools/gn`